helm/prometheus-meta-operator/templates/alertmanager/servicemonitor.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.alertmanager.enabled }}
```

Comment:
should we also remove alertmanager scraping from https://github.com/giantswarm/prometheus-meta-operator/blob/master/files/templates/scrapeconfigs/additional-scrape-configs.template.yaml#L660 ? 

this target is not matching any pods in discovery anyway , at least in CAPZ , because the label selector is not like this one in the SM

Reply:
However, i think we have to remove this target

Reply:
since, we have added a ServiceMonitor to scrape Alertmanager

Reply:
that's what i meant. since we are adding the servicemonitor then should we remove the hardcoded one ... did not have any better place to put that comment :)

Reply:
Thanks a lot